uvdat/core/rest/guardian.py

```diff
@@ -0,0 +1,51 @@
+from guardian.shortcuts import get_objects_for_user
+from rest_framework.filters import BaseFilterBackend
+from rest_framework.permissions import SAFE_METHODS, IsAuthenticated
+
+from uvdat.core import models
+
+
+class GuardianPermission(IsAuthenticated):
+    def get_object_queryset(self, obj):
+        if isinstance(obj, models.Project):
+            return obj
+        elif isinstance(models.Dataset):
```

Comment:
Just a general code style suggestion, the "else" part of the `elif` is never reached:
```suggestion
        if isinstance(obj, models.Project):
            return obj
        if isinstance(models.Dataset):
```

Reply:
Changed in [f831cf3](https://github.com/OpenGeoscience/uvdat/pull/60/commits/f831cf38eefd71fce0d36d418bb3e6810b6ef144)